soundcard/pulseaudio.py

```diff
@@ -63,7 +63,7 @@ def default_microphone():
         return get_microphone(name)
 
 
-def get_microphone(id, exclude_monitors=True):
+def get_microphone(id, include_loopback=False):
```

Comment:
We should still allow `exclude_monitors` for the time being, in parallel with `include_loopback`, but raise a `DeprecationWarning` if it is used, and convert it to the new `include_loopback`.